packages/web-component-tester/data/index.html

```diff
@@ -4,18 +4,15 @@
 <head>
   <meta charset="utf-8">
   <script>
-    WCT = <%= JSON.stringify(clientOptions) %>;
-  </script>
-  <script>
+    window.WCT = <%= JSON.stringify(clientOptions) %>;
     window.__generatedByWct = true;
   </script>
-  <script src="../<%= browserScript %>"></script>
-  <% extraScripts.forEach(function(script) { %>
-    <script src="<%- script %>"></script>
+  <% scripts.concat(extraScripts).forEach(function(href) { %>
+    <script src="<%= href %>"></script>
+  <% }); %>
+  <% modules.forEach(function(href) { %>
+    <script type="module" src="<%= href %>"></script>
   <% }); %>
-  <% if (typeof npm === 'undefined' || !npm) { %>
```

Comment:
what enable us to remove this?

Reply:
I think I posted this comment elsewhere, but I just essentially moved this responsibility out of the template logic and into an injection point here https://github.com/Polymer/tools/pull/706/files/502b65b0af88465b6a88019d61ffa9612d29f60b#diff-372709f429d9eea155388e6b408f3d59R143